tests/send_sync.rs

```diff
@@ -0,0 +1,22 @@
+winrt::import!(
+    dependencies
+        os
+    types
+        windows::foundation::Uri
+);
+
+use std::thread;
+use windows::foundation::Uri;
+
+// Simple test to validate that types with MarshalingType.Agile are marked Send and Sync
+// (if this compiles it worked)
+#[test]
+fn send_sync() -> winrt::Result<()> {
+    let url = Uri::create_uri("http://kennykerr.ca")?;
+
+    thread::spawn(move || {
+        println!("{:?}", url);
```

Comment:
Tests should `assert` rather than `println`. Also, can the test wait to ensure the assert in the thread has executed?